data_summaries/No Robots.json

```diff
@@ -0,0 +1,62 @@
+{
+    "no-robots": {
```

Comment:
Disentangle this PR from no-robots and privacy-200

Reply:
Fix this PR disentangle from privacy-200 and no-robot